aiida/engine/processes/control.py

```diff
@@ -0,0 +1,206 @@
+# -*- coding: utf-8 -*-
+"""Functions to control and interact with running processes."""
+from __future__ import annotations
+
+import concurrent
+import typing as t
+
+import kiwipy
+from kiwipy import communications
+from plumpy.futures import unwrap_kiwi_future
+
+from aiida.common.exceptions import AiidaException
+from aiida.common.log import AIIDA_LOGGER
+from aiida.manage.external.rmq import CommunicationTimeout
+from aiida.manage.manager import get_manager
+from aiida.orm import ProcessNode, QueryBuilder
+from aiida.tools.query.calculation import CalculationQueryBuilder
+
+LOGGER = AIIDA_LOGGER.getChild('process_control')
+
+
+class ProcessTimeoutException(AiidaException):
+    """Raised when action to communicate with a process times out."""
+
+
+def get_active_processes() -> list[ProcessNode]:
+    """Return all active processes, i.e., those with a process state of created, waiting or running.
+
+    :return: A list of process nodes of active processes.
+    """
+    filters = CalculationQueryBuilder().get_filters(process_state=('created', 'waiting', 'running'), paused=True)
+    builder = QueryBuilder().append(ProcessNode, filters=filters)
+    return builder.all(flat=True)
+
+
+def play_processes(
+    processes: list[ProcessNode] | None = None,
+    all_entries: bool = False,
+    timeout: float = 5.0,
+    wait: bool = False
+) -> None:
+    """Play (unpause) paused processes.
+
+    .. note:: Requires the daemon to be running, or processes will be unresponsive.
```

Comment:
I guess adding a check for the daemon status in the function would be expensive?

Reply:
That's why I didn't add it, but actually from the API it is quite fast
```
In [6]: %timeit get_daemon_client().is_daemon_running
63.1 µs ± 1.15 µs per loop (mean ± std. dev. of 7 runs, 10000 loops each)
```
So might as well add it.